crates/aptos/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to the Aptos CLI will be captured in this file. This project
 
 ## Unreleased
 
+## [4.2.2] - 2024/09/20
+- Fix localnet indexer processors that were emitting spamming logs in 4.2.1.
+
```

Comment:
Can you update `crates/aptos/Cargo.toml` for the new version

Reply:
done.